vendor/modules.txt

```diff
@@ -1,82 +1,63 @@
+# github.com/AdamKorcz/go-fuzz-headers v0.0.0-20210319161527-f761c2329661
+github.com/AdamKorcz/go-fuzz-headers
 # github.com/checkpoint-restore/go-criu/v4 v4.1.0
-## explicit
 github.com/checkpoint-restore/go-criu/v4
 github.com/checkpoint-restore/go-criu/v4/rpc
 # github.com/cilium/ebpf v0.2.0
-## explicit
```

Comment:
Not sure what you did with vendor/modules.txt but it seems wrong

Reply:
This was the result of running `make vendor`, which I understood will take care of everything when introducing a new dependency.

Reply:
That part is the result of running `make vendor` after having introduced a new dependency. Am I missing something or shouldn't `make vendor` handle new dependencies?